web/src/components/troubleshoot/GenerateSupportBundle.jsx

```diff
@@ -314,6 +314,37 @@ class GenerateSupportBundle extends React.Component {
                 </div>
               }
               {showRunCommand ?
+                watch.isAirgap ?
```

Comment:
we still calling this `watch`? :) wow

Reply:
haha yeah...still in the code here and there 🤡